### ISSUE TITLE
Popup's close button does not have sufficient contrast

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#7539]
## Motivation

The popup's close button has a contrast ratio of 1.76, and **fails** all of the following <span title="Web Content Accessibility Guidelines">WCAG 2.1</span> Success Criteria:
- [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
- [1.4.6 Contrast (Enhanced)](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
- [1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)

<img width="300" src="https://user-images.githubusercontent.com/26493779/113319315-9779c880-9311-11eb-90da-2e90f66c785b.png" alt="Popup contrast issue">

## Proposed solution

Increase the contrast of the popup's close button by defaulting to a darker color:

https://github.com/Leaflet/Leaflet/blob/b52961aa469dc17b3710ef7a9c0f13b6f4dba550/dist/leaflet.css#L521

## Additional context

Some browser's dev tools make suggestions for colors to increase the contrast to acceptable levels (Chrome in this case):

<img width="325" src="https://user-images.githubusercontent.com/26493779/113319359-a19bc700-9311-11eb-9c51-3d61dfe758c2.png" alt>

**Comment by Malvoz** [comment]
The color of the close button is a design choice, if it's decided not to change it for that reason then it could be added as a known limitation in the [accessibility statement](https://github.com/Leaflet/Leaflet/issues/7537).
